### PR TITLE
Silitski/hometask2

### PR DESCRIPTION
**Pull request by qqqwww123cdv**
Сompleted second homework.

### CODE REVIEW

File: hometask-2/get_middle_charachter.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def get_middle(str)
+  hf = str.length / 2
```
Comment (jekapulpit) [inline hometask-2/get_middle_charachter.rb]: let's try to name this variable more accurately? `hf` gives not many information 😄

File: hometask-2/get_middle_charachter.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  if str.length.even?
```
Comment (jekapulpit) [inline hometask-2/get_middle_charachter.rb]: we can use ternary operator there and your 5 lines will become 1

File: hometask-2/get_middle_charachter.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    str[hf - 1] + str[hf]
```
Comment (jekapulpit) [inline hometask-2/get_middle_charachter.rb]: and if you'll choose the ternary operator, the range looks better here `str[(hf - 1)..hf]`

**Review by jekapulpit — CHANGES_REQUESTED** [review]

**Review by ramantechart — CHANGES_REQUESTED** [review]
https://github.com/iTechArt-RubyLab/Ruby-Lab#%D0%B7%D0%B0%D0%B4%D0%B0%D0%BD%D0%B8%D0%B5-2

**Pull request closed without merging**